sample-demo-django-app/README.md

```diff
@@ -57,17 +57,31 @@ The main entry point is in ``sample-demo-django-app/mysite/wsgi.py`` where the c
     python3 manage.py loaddata initial_local_db_data.json
     ```
 
-2. Start the service using **one** of the following commands.
+2. Start the service using **one** of the following commands, depending on what web server you want.
+
+    2.1 Start the service using a development server.
     ```bash
-    # Start the service using a development server.
     python manage.py runserver
     ```
 
+    2.2. Start the service using [uWSGI](https://uwsgi-docs.readthedocs.io/en/latest/) with multiple workers; use the `--enable-threads and `--lazy-apps` parameters  for your startup configuration, so Profiler can be started. This was tested for uswgi version 2.0.19.1.
```

Comment:
I think you are missing a ` quote after `--enable-threads`

Reply:
Yes, will update, thanks.